std/stdio.d

```diff
@@ -1368,6 +1368,49 @@ Throws: $(D Exception) if the file is not opened.
         w.put('\n');
     }
 
+    /**
+
+    Writes its arguments in text format to the file separated pairwise by a
+    space (or custom `separator`), followed by a newline (or custom `eol`).
+
+    Throws: $(D Exception) if the file is not opened.
+            $(D ErrnoException) on an error writing to the file.
+    */
+    void print(string separator = " ", string eol = "\n", S...)(S args)
```

Comment:
Also, formatting a list of items delimited by some string isn't specific to file I/O; what about something in `std.format` that does this generically, returning an input range of `char` (or whatever) so that it can be reused for other stuff that doesn't involve file I/O?